Modules/Typescript/Design-Patterns/airEastShipper.ts

```diff
@@ -0,0 +1,17 @@
+import { ShipperWrapper, ShipmentTypeEnum } from './shipperWrapper';
+
+export class AirEastShipper extends ShipperWrapper {
+
+  constructor(weight: number) {
+    super(
+      'Air East',
+      ['1', '2', '3'],
+      {
+        [ShipmentTypeEnum.Letter]: 0.39,
+        [ShipmentTypeEnum.Package]: 0.25,
+        [ShipmentTypeEnum.Oversized]: 10,
```

Comment:
This is incorrect, you are saying that the formula is 10 * weight, but per task, it's 10 + .25 * weight